.github/ISSUE_TEMPLATE/issue.yaml

```
@@ -31,8 +33,41 @@ body:
       description: Contact and name info for key parties for the project.
       placeholder: |
         E.g.:
-        Casey Greene, PI, casey@greenelab.com, @caseygreene on slack
-        John Doe, project lead, john.doe@fake.com, @johndoe on slack
+        Casey Greene, PI, casey@greenelab.com, @caseygreene on Slack
+        John Doe, project lead, john.doe@fake.com, @johndoe on Slack
+    validations:
+      required: true
+
+  - type: dropdown
+    id: category
+    attributes:
+      label: Type of support
+      description: Which of these best describe the type of support you need?
+      options:
+        - Short/medium-term project (ongoing)
+        - Debugging/troubleshooting (one-off)
```

Suggesting the addition of `Support` here (sometimes I feel like folks might just want guidance on "am I doing this right").
```suggestion
        - Support/Debugging/troubleshooting (one-off)
```